common/styles/utilities/compiled_variables/_fonts.scss

```diff
@@ -25,7 +25,7 @@ $fonts: (
     'font-family-subset': "'Wellcome Bold Web Subset'",
     'font-family-web': "'Wellcome Bold Web'",
     'font-size': 33px,
-    'line-height': 48px
+    'line-height': 42px
```

Comment:
This belonged in the typography PR from a couple of days ago, but slipped in here.